emukit/core/loop/outer_loop.py

```diff
@@ -39,6 +41,7 @@ def __init__(self, candidate_point_calculator: CandidatePointCalculator, model_u
         self.loop_state = loop_state
         if self.loop_state is None:
             self.loop_state = LoopState([])
+        self.loop_iteration_end_event = EventHandler()
```

Comment:
Can you start a list in the docstring with events this object exposes? At the moment it will have just one item, but I am sure it will grow

Also I think "loop_" part can be dropped from the name, since it is a part of the loop object

Reply:
Done both items